server/models/players.js

```diff
@@ -1,23 +1,19 @@
-let mongoose = require('mongoose');
-let Schema = mongoose.Schema;
+const factory = require('./database.js');
 
-let PlayersSchema = new Schema({
+const schema = {
     name: String,
     nickname: String,
-    chant: String,
-    wins: Number,
-    losses: Number
```

Comment:
wins and losses didn't make the cut? was that intentional?

Reply:
Oh interesting, you know why that probably happened?  I started this, then got up to date after.  Good catch!